src/dnddiceroller/Die.java

```diff
@@ -1,12 +1,51 @@
 package dnddiceroller;
 
+import java.util.Random;
+
 /**
- * A Die is a many sided object that when rolled provides a random value from
- * 1 through the number of sides on the object. Some dice are 6 sided and have
- * the numbers 1-6 on them. Some dice are 20 sided with the numbers 1-20 on them.
- * Others are called fudge dice and have the values of -1 0 or +1
- * @author Paul Scarrone
+ *
+ * @author Rhonda Marshall
+ * Oct 19, 2015
  */
 public class Die {
-  
-}
+    public int numberOfSides;             // number of sides on the dice
+    public int value;                     // value generated by a random number
```

Comment:
Yes, Once AGAIN!! I have forgotten to make these private
 
 
-----Original Message-----
From: Paul Scarrone <notifications@github.com>
To: WCCCEDU/CPT-163-27-F2015-Assignment-7-Array-DiceTower <CPT-163-27-F2015-Assignment-7-Array-DiceTower@noreply.github.com>
Cc: Rhonda R. Marshall <rhnmrsh@aol.com>
Sent: Wed, Oct 28, 2015 8:25 am
Subject: Re: [CPT-163-27-F2015-Assignment-7-Array-DiceTower] Assignment 7, Arrays (#8)



In src/dnddiceroller/Die.java:
>   */
>  public class Die {
> -  
> -}
> +    public int numberOfSides;             // number of sides on the dice
> +    public int value;                     // value generated by a random number

Only final fields should ever be public and in most cases they still should be private. We do not want to allow external modification of these after the class instance has been initialized otherwise the class could lose its context without us being able to track that modification.
—
Reply to this email directly or view it on GitHub.